civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +72,194 @@
     }
   }
 }
+
+.view-mydetails-personal {
+  .chr_panel--my-details__personal__contact_id span {
+    @include views-label;
+  }
+
+  .pane-block {
+    border-bottom: 1px solid $gray-light;
+    margin-bottom: 0;
+  }
+
+  .views-row {
+    min-height: 80px;
+    padding-left: 0;
+    position: relative;
+
+    @media (min-width: 320px) {
+      padding-left: 100px;
+    }
+  }
+
+  .views-field-image-URL {
+    display: none;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 80px;
+
+    @media (min-width: 320px) {
```

Comment:
What exactly is this breakpoint for?

Reply:
To show profile picture for devices with 320px or more.